app/src/main/res/layout-sw320dp/activity_description.xml

```diff
@@ -277,7 +277,8 @@
                             android:fontFamily="@font/armata"
                             android:onClick="registerToActivity"
                             android:text="@string/activity_description_register_button"
-                            tools:ignore="ButtonStyle" />
+                            tools:ignore="ButtonStyle"
```

Comment:
Do you add the space around words ?

Reply:
Oui, c'est plus propre sur la description de l'activité maintenant !

Reply:
Yes, it's cleaner on the activity description now!